pipeline_dp/dp_computations.py

```diff
@@ -285,9 +286,8 @@ def compute_dp_sum(sum: float, dp_params: MeanVarParams):
 
 
 def _compute_mean(
-    count: float,
     dp_count: float,
-    sum: float,
+    normalized_sum: float,
```

Comment:
We changed behaviour of this function and expectations on the arguments. Let's reflect that in function signature/comments.

1. Maybe something like `_compute_mean_for_normalized_sum` (and to name the argument `sum`)?
2. After normalization all elements are in the symmetric interval, maybe to use one arg `max_abs_value` instead of `min_value/max_value`